libopae/api-shell.c

```diff
@@ -1637,37 +1647,32 @@ fpga_result fpgaDestroyFeatureToken(fpga_feature_token *token)
 	wrapped_token = opae_validate_wrapped_feature_token(*token);
 
 	ASSERT_NOT_NULL(wrapped_token);
-	
+
 	ASSERT_NOT_NULL_RESULT(wrapped_token->adapter_table->fpgaDestroyFeatureToken,
```

Comment:
Does this macro cause the function to return without freeing the `wrapped_token`? If so, would we leak memory here?